classes/settings.php

```diff
@@ -10,9 +10,9 @@
 }
 
 /**
- * Class for Dintero settings
+ * Class for plugin settings.
  */
-class Zaver_Checkout_Settings {
+class Settings {
```

Comment:
This class does not have any namespace. So it will be in the global namespace with just the name Settings. That is very likely to cause a potential issue. Add the namespace similar to any other class in this folder, and make sure all use statements are updated.